homeassistant/components/proxmoxve/sensor.py

```diff
@@ -0,0 +1,133 @@
+"""Sensor to read Proxmox VE data."""
+
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+)
+from homeassistant.const import UnitOfInformation
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from . import COORDINATORS, DOMAIN, PROXMOX_CLIENTS, ProxmoxEntity
+
+BYTE_TO_GIBIBYTE = 1.074e9
+
+
+@dataclass
+class ProxmoxVERequiredKeyMixin:
+    """Mixin for required lambda functions."""
+
+    native_lambda: Callable
```

Comment:
Please improve type hints here.
```suggestion
    native_lambda: Callable[[ProxmoxAPIData], int]
```